packages/ref-metrics-esmvaltool/src/ref_metrics_esmvaltool/recipe.py

```diff
@@ -0,0 +1,209 @@
+from __future__ import annotations
+
+import subprocess
+from pathlib import Path
+from typing import TYPE_CHECKING, Any
+
+import pkg_resources
```

Comment:
pkg_resources is deprecated. Is it possible to replace this usage with `importlib.resources`?

Reply:
Thanks for flagging that. I will update it and opened an issue to report this to the authors of pooch: https://github.com/fatiando/pooch/issues/448.

Reply:
Done in 5cdaa67